### DISCUSSION TITLE
Custom Tabs and futures options within the Overseerr sidemenu

### DISCUSSION THREAD

**Discussion by Dabu-dot** [sct/overseerr#594] [Feature Requests]
So this discussion is only about sharing potentials future utilisations and plans within the Overseer Menu (that's mainly why I didn't opened a feature request about it as I want to hear opinions on it). Right now the Overseerr menu look like this:

![Screenshot_10](https://user-images.githubusercontent.com/52525576/103789447-4d4db780-5040-11eb-912c-2f09bcae130a.jpg)

It has all the functionalities that are essential to the core of Overseerr but a good chunk of this space could still be used for other purposes. 

Custom tabs to redirect to Plex could exist (or others small apps - [MovieMatch](https://github.com/LukeChannings/moviematch) just came out and could be linked pretty easily with Overseerr using a custom tab). 

User Watchlist could be also listed there.

Ombi use differents tabs linked to Media Management as for "Issues" notices, "Votes" for medias getting a minimal requests numbers before sending it to an Arr app, "Donation" and "Custom Donation" link etc..

![Screenshot_11](https://user-images.githubusercontent.com/52525576/103791228-7cfdbf00-5042-11eb-9c71-56c7e7ba8e53.jpg)

Does any of these ideas interest you guys or is expanding the sidemenu with features requests is simply not a thing considered for now ?

**Comment by ghost** [comment]
Would love to see a custom donation button 👍

**Comment by wearethedynamite** [comment]
+1 for custom donate button, love this so far, this is the only thing stopping me from fully migrating from Ombi, 👍

**Comment by arjohnson15** [comment]
+1 for a couple buttons that can be used as redirects.

**Comment by horrorshow21** [comment]
custom donate button for sure!

**Comment by jfresq** [comment]
+1 for custom button options
